plugins/httpjson/httpjson.go

```diff
@@ -126,7 +133,7 @@ func (h *HttpJson) gatherServer(acc plugins.Accumulator, service Service, server
 		return err
 	}
 
-	var jsonOut interface{}
+	var jsonOut map[string]interface{}
```

Comment:
Won't this break the entire plugin if there is a non-string key in the top-level of the JSON?